services/app-web/src/common-code/proto/healthPlanFormDataProto/unlockedWithALittleBitOfEverything.test.ts

```diff
@@ -122,11 +122,6 @@ describe('0001_rate_id_migration', () => {
             throw migratedFormData
         }
 
-        //Rate certification name generation is a part of a different migration, so to test our rate id migration we will
-        // manually generate a rate id.
-        migratedFormData.rateInfos[0].rateCertificationName =
-            'MCR-MN-0005-SNBC-RATE-20220621-20221021-AMENDMENT-20210523'
```

Comment:
Oh we were able to remove this once you merged main? Can you share more about what happened here because I was going to write a ticket about this pattern today. Is it not needed then to do these kind of workarounds in proto tests generally?

Reply:
Yes! It was actually @rswerve rate certification name versioning work that made old data symmetrical with new domain model. Prior, old data did not have rate certification and once you encode, decode and convert to domain model a certification name was generated and now is asymmetrical.